### ISSUE TITLE
Clarification on the DeiT-3 checkpoint provided

### ISSUE THREAD

**Issue by jiyunBae007** [kirill-vish/Beyond-INet#2]
Hi,

I would like to clarify whether the DeiT-3 checkpoint provided is indeed "pretrained on ImageNet-21K and fine-tuned on ImageNet-1K," as mentioned in the README.

I ran the shape_texture.py script and compared the results with the checkpoint provided at https://dl.fbaipublicfiles.com/deit/deit_3_base_21k.pth, which is only pretrained on ImageNet-21K without fine-tuning. Surprisingly, both this checkpoint and the provided DeiT-3 checkpoint yielded the same results in terms of accuracy and shape bias, as shown in the attached image.

Could you confirm if the provided checkpoint has been fine-tuned on ImageNet-1K, or is it also only pretrained on ImageNet-21K?

Thank you for your help!
<img width="919" alt="result" src="https://github.com/user-attachments/assets/a280dcb8-f26f-4cfa-9bd1-c1fb7227d85b">

**Comment by kirill-vish** [comment]
Hey, I cannot be 100% sure since I didn't train these models, but Table 9 in DeiT3 paper (https://arxiv.org/abs/2204.07118) indicates that the authors did 50 epoch finetuning on ImageNet-1K after pretraining on ImageNet-21k. 
![Screenshot 2024-09-11 at 11 50 15 PM](https://github.com/user-attachments/assets/fdcaf31a-22aa-4a60-bc1f-6dfff7356b35)


They also have the following text on page 14 in their paper:

> ImageNet-21k. In Table 8 we compare ViT architecture pre-trained on ImageNet21k with our training recipe then finetuned on ImageNet-1k. We can observe that the findings are similar to what we obtained on ImageNet-1k only.

So based on that I assume that this checkpoint should be finetuned on ImageNet-1K in the end, even though it is not specifically mentioned in the name of the checkpoint.

**Comment by jiyunBae007** [comment]
Thanks for the quick response. really appreciate it!
